app/views/create_user.py

```diff
@@ -0,0 +1,49 @@
+from flask import session, redirect, url_for, request, render_template, flash
+from app import app, forms
+from app.util import is_logged_in
+import pymesync
+
+
+@app.route('/users/create', methods=['GET', 'POST'])
+def create_user():
+    # Check if logged in first
+    if not is_logged_in():
+        if request.method == 'GET':
+            return redirect(url_for('login', next=request.url_rule))
+        elif request.method == 'POST':
+            return "Not logged in.", 401
+
+    form = forms.CreateUserForm()
+
+    ts = pymesync.TimeSync(baseurl=app.config['TIMESYNC_URL'],
+                           test=app.config['TESTING'],
+                           token=session['token'])
+
+    # If form submitted (POST)
+    if form.validate_on_submit():
+        user = {}
+
+        for field in form:
```

Comment:
Ah, I see, it's from pymesync - yes, we'll have to strip that field out before sending it to pymesync.